module/VuFindTheme/src/VuFindTheme/View/Helper/HeadTitlewithInformation.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Head Title with additional Information View Helper
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2010.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Gregor Gawol <gawol@ub.uni-leipzig.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+namespace VuFindTheme\View\Helper;
+
+/**
+ * Head Title with additional Information View Helper
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Gregor Gawol <gawol@ub.uni-leipzig.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class HeadTitlewithInformation extends \Laminas\View\Helper\AbstractHelper
```

Comment:
I wonder if AugmentedHeadTitle or ExpandedHeadTitle might be a more concise name (would also potentially clear up the alias inconsistency I noted above).